internal/imap_filter/command/command.go

```diff
@@ -127,15 +128,23 @@ func (c *Check) expandCommand(msgMeta *module.MsgMetadata, accountName string) (
 				valI, err := msgMeta.Conn.RDNSName.Get()
 				if err != nil {
 					return ""
-				}
+				}x
 				if valI == nil {
-					return ""
+					return ""x
 				}
 				return valI.(string)
 			case "{msg_id}":
 				return msgMeta.ID
 			case "{sender}":
 				return msgMeta.OriginalFrom
+			case "{rcpts}":
```

Comment:
I wonder if it is a better idea to name this original_rcpts to avoid confusion.

Reply:
please see my reply bellow - it looks like OriginalRcpts is not set when the Rcpts is not replaced. Will need to fix that. Then I will rename to originalRcpts as proposed.

Reply:
renamed. Also ensured all original rcpts are added and not only those modified by the msgpipeline